File: hw/manosalvas/U1/HW09/CondoManage/src/ec/espe/edu/condomanage/model/ParkingLot.java
package ec.espe.edu.condomanage.model;

public class ParkingLot {
    private int spaceNumber;
    private boolean isOccupied;

    public ParkingLot(int spaceNumber, boolean isOccupied) {
        this.spaceNumber = spaceNumber;
        this.isOccupied = isOccupied;
    }

    // Getters y Setters
    public int getSpaceNumber() {
        return spaceNumber;
    }

    public void setSpaceNumber(int spaceNumber) {
        this.spaceNumber = spaceNumber;
    }

    public boolean isOccupied() {
        return isOccupied;
    }

    public void setOccupied(boolean isOccupied) {
        this.isOccupied = isOccupied;
    }

    @Override
    public String toString() {
        return "ParkingLot{" +
                "spaceNumber=" + spaceNumber +
                ", isOccupied=" + isOccupied +
                '}';
    }
}

File: hw/manosalvas/U1/HW08ObjectsDefinningAttributesMethods/CondoManagepy/parking_lot.py
# parking_lot.py
class ParkingLot:
    def __init__(self, space_number, is_occupied):
        self.space_number = space_number
        self.is_occupied = is_occupied
    
    def __str__(self):
        return f"ParkingLot(space_number={self.space_number}, is_occupied={self.is_occupied})"
